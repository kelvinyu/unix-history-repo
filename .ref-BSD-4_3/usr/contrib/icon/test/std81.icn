record array(a,b,c,d,e,f,g)

procedure dummy(u,v,x,y,z)
   suspend u | v
   return x
end

procedure main()
   write("c1 := 'abcde' ----> ",image(c1 := 'abcde') | "none")
   write("c2 := 'aeuoi' ----> ",image(c2 := 'aeuoi') | "none")
   write("c1 ++ c2 ----> ",image(c1 ++ c2) | "none")
   write("c1 -- c2 ----> ",image(c1 -- c2) | "none")
   write("c1 ** c2 ----> ",image(c1 ** c2) | "none")
   write("[] ** \"abc\" ----> ",image([] ** "abc") | "none")
end
