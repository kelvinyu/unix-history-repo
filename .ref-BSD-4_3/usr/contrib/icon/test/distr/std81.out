c1 := 'abcde' ----> 'abcde'
c2 := 'aeuoi' ----> 'aeiou'
c1 ++ c2 ----> 'abcdeiou'
c1 -- c2 ----> 'bcd'
c1 ** c2 ----> 'ae'
Run-time error 104 at line 14 in std81.icn
cset expected
offending value: []
